package_manifest.yaml

```diff
@@ -0,0 +1,415 @@
+packages:
+  children:
+    foreman: {}
+    foreman-nonscl: {}
+    plugins: {}
+    plugins-nonscl: {}
+    katello: {}
+
+foreman:
+  vars:
+    package_base_dir: 'packages/foreman/'
+    releasers:
+    - koji-foreman-nightly
+  hosts:
+    foreman: {}
+    rubygem-activerecord-session_store: {}
+    rubygem-addressable: {}
+    rubygem-algebrick:
+      releasers:
+      - koji-foreman-nightly
+      - koji-foreman-plugins-nightly
+    rubygem-ancestry: {}
+    rubygem-apipie-bindings: {}
+    rubygem-apipie-params:
+      releasers:
+      - koji-foreman-nightly
+      - koji-foreman-plugins-nightly
+    rubygem-apipie-rails: {}
+    rubygem-audited: {}
+    rubygem-autoparse: {}
+    rubygem-autoprefixer-rails: {}
+    rubygem-awesome_print: {}
+    rubygem-bootstrap-sass: {}
+    rubygem-bundler_ext: {}
+    rubygem-clamp: {}
+    rubygem-concurrent-ruby:
+      releasers:
+      - koji-foreman-nightly
+      - koji-foreman-plugins-nightly
+    rubygem-concurrent-ruby-edge:
+      releasers:
+      - koji-foreman-nightly
+      - koji-foreman-plugins-nightly
+    rubygem-css_parser: {}
+    rubygem-daemons: {}
+    rubygem-deacon: {}
+    rubygem-deep_cloneable: {}
+    rubygem-domain_name: {}
+    rubygem-dynflow:
+      releasers:
+      - koji-foreman-nightly
+      - koji-foreman-plugins-nightly
+    rubygem-eventmachine: {}
+    rubygem-excon: {}
+    rubygem-extlib: {}
+    rubygem-facter: {}
+    rubygem-faraday:
+      releasers:
+      - koji-foreman-nightly
+      - koji-foreman-plugins-nightly
+    rubygem-fast_gettext:
+      releasers:
+      - koji-foreman-nightly
+      - koji-foreman-plugins-nightly
+    rubygem-fog: {}
+    rubygem-fog-aws: {}
+    rubygem-fog-core: {}
+    rubygem-fog-digitalocean: {}
+    rubygem-fog-google: {}
+    rubygem-fog-json: {}
+    rubygem-fog-libvirt: {}
+    rubygem-fog-openstack: {}
+    rubygem-fog-ovirt: {}
+    rubygem-fog-rackspace: {}
+    rubygem-fog-vsphere: {}
+    rubygem-fog-xenserver: {}
+    rubygem-fog-xml: {}
+    rubygem-font-awesome-sass: {}
+    rubygem-formatador: {}
+    rubygem-friendly_id: {}
+    rubygem-get_process_mem: {}
+    rubygem-gettext: {}
+    rubygem-gettext_i18n_rails: {}
+    rubygem-gettext_i18n_rails_js: {}
+    rubygem-google-api-client: {}
+    rubygem-gridster-rails: {}
+    rubygem-hammer_cli: {}
+    rubygem-hammer_cli_foreman: {}
+    rubygem-hashie: {}
+    rubygem-highline: {}
+    rubygem-hirb: {}
+    rubygem-hirb-unicode-steakknife: {}
+    rubygem-hoe: {}
+    rubygem-http-cookie: {}
+    rubygem-ipaddress: {}
+    rubygem-jquery-turbolinks: {}
+    rubygem-jquery-ui-rails: {}
+    rubygem-jquery_pwstrength_bootstrap_4: {}
+    rubygem-jwt: {}
+    rubygem-launchy: {}
+    rubygem-ldap_fluff: {}
+    rubygem-little-plugger: {}
+    rubygem-locale: {}
+    rubygem-logging: {}
+    rubygem-multi_json: {}
+    rubygem-multipart-post: {}
+    rubygem-mysql2:
+      releasers:
+      - koji-foreman-nightly
+      - koji-foreman-plugins-nightly
```

Comment:
This is a good red-flag to find! This should not exist in both of these releasers anymore. The `koji-foreman-plugins-nightly` should be dropped since this is in Foreman core repositories now.